src/parser.rs

```diff
@@ -1547,7 +1523,7 @@ expected_indicator:    {}
         #[test]
         fn test_block_comment(
             block_comment_content in r#"([^*]|([*][^*/]))*"#,
```

Comment:
`#` characters? I'm not sure why they are removed in many places, but not everywhere that doesn't have embedded doublequotes. (I prefer the consistency of always having them, but if they are going to be removed they should be removed consistently.)